src/diff/children.js

```diff
@@ -255,11 +255,19 @@ export function toChildArray(children, callback, flattened) {
 		}
 	} else if (!callback) {
 		flattened.push(children);
-	} else if (typeof children == 'string' || typeof children == 'number') {
-		flattened.push(callback(createVNode(null, children, null, null)));
+	} else if (typeof children === 'string' || typeof children === 'number') {
```

Comment:
TODO: correct this, must've been a merge conflict

Reply:
```suggestion
	} else if (typeof children == 'string' || typeof children == 'number') {
```
Like this?